polaris-react/scripts/utilities/getCustomPropertyNames.js

```diff
@@ -31,12 +31,17 @@ const getGroupedCustomPropertyNames = () => {
       const tokenGroup = require(path.join(tokenGroupsDir, fileName));
 
       let tokenGroupName = path.basename(fileName, '.json');
-      if (fileName.startsWith('color.')) {
+      // rename the color.light tokens
+      if (fileName.includes('.light')) {
         tokenGroupName = 'color';
       }
-      const customPropertyNames = Object.keys(tokenGroup).map(
-        (token) => `--p-${token}`,
-      );
+      const customPropertyNames = Object.keys(tokenGroup).map((token) => {
+        return {
+          label: token,
```

Comment:
```suggestion
          name: token,
```
We've been loosely following the Design Token Format Module terminology. `Token Group`, `Token Name` and `Token Alias`: https://design-tokens.github.io/community-group/format/#example-14